src/tests/test_utils.py

```diff
@@ -704,14 +707,30 @@ def establish_cis_connection(transport, central, peripheral, trace, params, acl_
     return success, cisConnectionHandles
 
 
+def calc_conn_supervision_timeout_ms(iso_interval_ms):
+    """
+    Calculate the minimum ACL supervision timeout that can be used to create a CIG with given ISO_Interval
+    :param iso_interval: ISO_Interval
+    :return: connSupervisionTimeout
+    """
+    # According to Core 5.3 | Vol 6, Part B 4.5.13.2 CIS events and subevents:
+    # "ISO_Interval (...) shall be less than half the connSupervisionTimeout for the associated ACL."
```

Comment:
LL.TS also states that "The connSupervisionTimeout for an ACL with associated CISes shall be greater than twice that of the ISO_Intervals of the associated CISes." (i.e., this PR reflects the intention of the test specification).

Reply:
+1. I'll change the quote. I missed this sentence in TS, thus I was looking in Core

Reply:
Either quote is fine, the comment was just to emphasize that the change is good :-)